apps/etl-func/src/domain/interfaces/tokenizer.ts

```diff
@@ -0,0 +1,20 @@
+import { z } from "zod";
+
+export interface TokenizerClient {
+  maskSensitiveInfo(pii: string): Promise<string>;
+}
+
+const messageWithoutPIISchema = z.object({
+  fiscalCode: z.string().uuid().min(1),
+});
+
+export type MessageWithoutPII = z.infer<typeof messageWithoutPIISchema>;
+
+export const tokenize =
+  (message: { fiscalCode: string }) =>
```

Comment:
```suggestion
  (message: Message) =>
```

Reply:
I had to use the MessageMetaData type instead